documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
+
+Additional capital contributions can be voted on and approved by the
+Corporate Overlords during a plotting session.
+
+Question for Howard: how do we allow members *or* CO's to contribute
+funds directly to specific BIFFUD projects without having equity impact?
+
+
+### Allocating Resources to Projects
+The Corporate Overlords can vote to assign budgets to BIFFUD Projects
+which are then overseen by the project owners.  The project owners
+must select a single person to be the project's "Cashier" who is
```

Comment:
Renaming this to Comptroller since that was a good one -- we should add this to Section 2 under "Project Roles" <-- it may be the only project role required.